src/extension/serializer.ts

```diff
@@ -474,36 +478,35 @@ export class WasmSerializer extends SerializerBase {
 }
 
 export class GrpcSerializer extends SerializerBase {
```

Comment:
@hotpocket The existing abstraction that elect to run the serializer via GRPC or WASM (legacy; should be removed). Ideally we'd have a `BufEsSerializer` or something like that which makes it possible to run the serializer using a feature flag.

I'm somewhat uncomfortable making a hard cutover because it requires front-loading of extensive testing which is very difficult to coordinate.